helm/flowforge/values.yaml

```
@@ -36,6 +36,13 @@ forge:
   support:
     enabled: false
 
+  rate_limits:
+    enabled: false
+    global: false
+    max: 1000
+    maxAnonymous: 10
+    timeWindow: 60000
```

Should these not match the public defaults we decided in the issue Ben?

If yes, then they should be...

 ```suggestion
  rate_limits:
    enabled: false - disabled by default.
    # global: true - A setting to enable limits across all routes
    # max: 1000 - the default rate limit is 1000 operations per route over the time window
    # timeWindow: '1 Minute' - or 60000
```

---

These values need to be fully defined in this file, we can't comment them out, and `timeWindow` must be a number.

I've pushed an update to change global to true